src/components/CardSlider/Card/index.tsx

```diff
@@ -0,0 +1,57 @@
+import { Box, Badge, Image } from '@chakra-ui/react'
+import { MdEvent } from 'react-icons/md'
+
+export const Card = ({ event }) => {
+  return (
+    <Box
+      maxW="sm"
+      boxShadow="lg"
+      borderRadius="lg"
+      overflow="hidden"
+      backgroundColor="#fff"
+      cursor="pointer"
+      spacing={2}
+      _hover={{
+        transform: 'scale(1.04)',
+        transition: '0.2s ease-out',
+      }}
+    >
+      <Box>
+        <Image
+          src={event.imageUrl}
+          alt={event.imageAlt}
+          width="100%"
+          height="150px"
+        />
+      </Box>
+
+      <Box p="4" display="flex" flexDirection="column" alignItems="flex-start">
+        <Box mt={1}>
+          <Badge
+            borderRadius="sm"
+            px={2}
+            py={0.5}
+            color="white"
+            bgGradient="linear(to-tl, orange.300, orange.500)"
+          >
+            EVENT
```

Comment:
It should be dynamic according to the type of the data:
`EVENT | ANNOUNCEMENT | COMPETITION | NEWS ...`